tests/components/shelly/test_sensor.py

```diff
@@ -358,6 +358,23 @@ async def test_block_sensor_unknown_value(
     assert hass.states.get(entity_id).state == STATE_UNKNOWN
 
 
+async def test_block_sensor_unknown_option_value(
```

Comment:
This test is almost a copy of https://github.com/home-assistant/core/blob/788cbc3e444576e319554e901cee24b449bd6452/tests/components/shelly/test_sensor.py#L348 just with a different sensor, it would be better to parameterize this test to cover the sensor options ("warmup", "normal", "fault") and give it a name that match the sensor it is testing

Reply:
Improved